### PR TITLE
Fixes a runtime in /datum/preference/New().

### PR DESCRIPTION
**Pull request by MarinaGryphon**
Moves `update_preview_icon()` to after `client` has actually been assigned.

### CODE REVIEW

**Comment by MarinaGryphon** [comment]
!bugfix

**Comment by MarinaGryphon** [comment]
!review

File: html/changelogs/MDP-fixes-prefs-runtime-1.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+################################
```
Comment (MarinaGryphon) [inline html/changelogs/MDP-fixes-prefs-runtime-1.yml:1]:
```suggestion
################################ 
```

File: html/changelogs/MDP-fixes-prefs-runtime-1.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
```
Comment (MarinaGryphon) [inline html/changelogs/MDP-fixes-prefs-runtime-1.yml:1]:
```suggestion
################################
```

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Pull request merged**